source/treeInterceptorHandler.py

```diff
@@ -144,22 +144,27 @@ def makeTextInfo(self,position):
 
 class RootProxyTextInfo(textInfos.TextInfo):
 
-	def __init__(self,obj,position,_rangeObj=None):
+	def __init__(self,obj,position,**kwargs):
 		super(RootProxyTextInfo,self).__init__(obj,position)
-		self.innerTextInfo=self.InnerTextInfoClass(obj.rootNVDAObject,position,_rangeObj=_rangeObj)
-
-	def _get_InnerTextInfoClass(self):
-		return self.obj.rootNVDAObject.TextInfo
+		self.InnerTextInfoClass=self.obj.rootNVDAObject.TextInfo

```

Comment:
I'm concerned this will break Word, which seems to set InnerTextInfoClass to WordDocumentTextInfoForTreeInterceptor, which overrides shouldIncludeLayoutTables.